dx_app_code/parliament2/parliament2.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+import dxpy
+import subprocess
+import os.path
+import glob
+
+@dxpy.entry_point('main')
+def main(**job_inputs):
+    # Preparation
+    sv_caller_results = []
+    
+    if 'illumina_bai' in job_inputs:
+        illumina_bai = job_inputs['illumina_bai']
+
+    if 'prefix' not in job_inputs:
```

Comment:
@slzarate this `if` needs and `else` that sets `prefix = job['prefix']`

If this will be/is the parliament2 code

Reply:
It is, thanks for pointing this out.